design-patterns/gc-intro/index.json-ld

```diff
@@ -0,0 +1,220 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Introduction block - Documentation and working example",
+		"fr": "Bloc d’introduction - Documentation et example pratique"
+	},
+	"description": {
+		"en": "Guidance about using the introduction block pattern on Canada.ca.",
+		"fr": "Orientation sur l’utilisation de la configuration de conception du bloc d’introduction sur Canada.ca."
+	},
+	"modified": "2023-08-16",
+	"componentName": "gc-intro",
+	"pages": {
+		"docs": [
+			{
+				"title": "Introduction block",
+				"language": "en",
+				"path": "gc-intro-doc-en.html"
+			},
+		{
+				"title": "Bloc d’introduction",
+				"language": "fr",
+				"path": "gc-intro-doc-fr.html"
+			}
+		],
+		"examples": [
+			{
+				"title": "Introduction block",
+				"language": "en",
+				"path": "gc-intro-en.html"
+			},
+			{
+				"title": "Bloc d’introduction",
+				"language": "fr",
+				"path": "gc-intro-fr.html"
+			},
+			{
+				"title": "Introduction block with image",
+				"language": "en",
+				"path": "gc-intro-image-en.html"
+			},
+			{
+				"title": "Bloc d’introduction avec image",
+				"language": "fr",
+				"path": "gc-intro-image-fr.html"
+			}
+		]
+	},
+	"dependencies": [
+		"bg-cover"
```

Comment:
Don't need to list the CSS utilities, but it worth to list the component/plugin because that support the intended behaviour of this design pattern.

```suggestion
		"data-bgimg-srcset"
```